integrations/influxdb_connector/influxdb-timestream-connector/README.md

```diff
@@ -95,6 +95,8 @@ The following parameters are available when deploying the connector as part of a
 | `CustomPartitionKeyDimension` |  The dimension to use as the partition key. This parameter is required if the CustomPartitionKeyType parameter is set to `dimension`. | |
 | `CustomPartitionKeyType` | The type of custom partition key to use. Valid options are `dimension` or `measure`. The `dimension` option requires the CustomPartitionKeyDimension parameter to also be set. If this parameter is not provided, newly-created tables will use default partitioning and none of the parameters relating to custom partition keys will be used. | |
 | `DatabaseName`  | The name of the database to use for ingestion. | `influxdb-line-protocol` |
```

Comment:
There are two table mapping modes for the InfluxDB Timestream Connector, `single-table` and `multi-table`. All data is ingested to the database set with the environment variable `DatabaseName`. If we use `multi-table` mapping, tables are created dependent on the `measurement` in the line-protocol. If we use `single-table` mapping, all data is ingested into the table set with the `SingleTableName` environment variable.

If we support database tags for database creation, we should also support table tags when `TableMapping` is `single-table`, and table creation is enabled.